server/README.md

```diff
@@ -59,6 +59,9 @@ This demo implementation logs a message to the demo channel whenever a user leav
 This demo implementation responds to a `/demo_plugin` command, allowing the user to enable
 or disable the demo plugin's hooks functionality (but leave the command and webapp enabled).
 
+A third option 'ephemeral' demonstrates ephemeral interactive usage of SendEphemeralPost, 
```

Comment:
This has to be updated. Sorry, forgot it in the last review.

Reply:
Done.  👍